src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="group group-pesonal">
+        <legend>Personal information</legend>
+        <label
+          class="label"
+          for="surname"
+        >
+          Surname:
+          <input
+            class="input"
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label
+          class="label"
+          for="name"
+        >
+            Name:
+          <input
+            class="input"
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label
+          class="label"
+          for="age"
+        >
+            How old are You?
+          <input
+            class="input"
+            type="number"
+            id="age"
+            value="12"
+            name="age"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label
+          class="label"
+          for="birth"
+        >
+            Full date of birth:
+          <input
+            class="input"
+            type="date"
+            id="birth"
+            placeholder="mm/dd/yyyy"
+            name="birth"
+          >
+        </label>
+        <label
+          class="label"
+          for="accept"
+        >
+            I accept the term of the agreement
+          <input
+            class="input"
+            type="checkbox"
+            id="accept"
+            name="accept"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group group-reg">
+        <legend>Registration</legend>
+        <label
+          class="label"
+          for="email"
+        >
+            E-mail:
+          <input
+            class="input"
+            name="email"
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label
+          class="label"
+          for="password"
+        >
+            Password:
+          <input
+            class="input password"
+            type="password"
+            id="password"
+            name="password"
+            minlength="5"
+            maxlength="15"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group group-fact">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label class="label">
+            Do you love cats?
+          <label
+            for="y"
+          >
+            <input
+              class="input"
+              type="radio"
+              id="y"
```

Comment:
same here, think of naming. You want your code to be maximally clear and readable